Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "autoclockspeed"
 version = "0.1.10"
 edition = "2018"
-license = "MIT"
+license = "MIT OR MPL-2.0"
```

Comment:
Do we know if crates.io supports this type of field?

Reply:
yep, i saw a few projects do that in their manifest

Reply:
Sweet, thanks so much